phpseclib/File/ANSI.php

```diff
@@ -213,6 +181,16 @@ class File_ANSI
      */
     function File_ANSI()
     {
+        $attr_cell = new stdClass();
+        $attr_cell->bold = false;
+        $attr_cell->underline = false;
+        $attr_cell->blink = false;
+        $attr_cell->background = 'black';
+        $attr_cell->foreground = 'white';
+        $attr_cell->reverse = false;
+        $this->base_attr_cell = clone($attr_cell);
```

Comment:
We need to update https://github.com/phpseclib/phpseclib/blob/1.0/composer.json#L60 then.